Lombiq.HelpfulLibraries/Libraries/Users/ICachingUserManager.cs

```diff
@@ -10,28 +10,36 @@ namespace Lombiq.HelpfulLibraries.Libraries.Users
     public interface ICachingUserManager
     {
         /// <summary>
-        /// Retrieves <see cref="User"/>s from a transient per-request cache by their unique ID or gets them from the
-        /// store if not yet cached.
+        /// Retrieves <see cref="User"/>s from a transient per-request cache by their <see cref="User.Id"/> or gets them
+        /// from the store if not yet cached.
+        /// </summary>
+        /// <param name="id">Unique ID identifying the <see cref="User"/> document.</param>
+        /// <returns>Potentially cached <see cref="User"/>.</returns>
+        Task<User> GetUserByIdAsync(string id, bool forceUpdate = false);
+
+        /// <summary>
+        /// Retrieves <see cref="User"/>s from a transient per-request cache by their <see cref="User.Id"/> or gets them
+        /// from the store if not yet cached.
         /// </summary>
         /// <param name="userId">Unique ID identifying the <see cref="User"/>.</param>
         /// <returns>Potentially cached <see cref="User"/>.</returns>
-        Task<User> GetUserByIdAsync(string userId);
+        Task<User> GetUserByUserIdAsync(string userId, bool forceUpdate = false);
 
         /// <summary>
         /// Retrieves <see cref="User"/>s from a transient per-request cache by their username or gets them from the
         /// store if not yet cached.
         /// </summary>
         /// <param name="username">Username of the <see cref="User"/>.</param>
         /// <returns>Potentially cached <see cref="User"/>.</returns>
-        Task<User> GetUserByNameAsync(string username);
+        Task<User> GetUserByNameAsync(string username, bool force = false);

```

Comment:
Why is the parameter called `force` here, not `forceUpdate`?

Reply:
Originally I called all just force, but then renamed to forceUpdate to be clearer. I guess I missed this one.